spec/api_manager_helper.rb

```diff
@@ -0,0 +1,21 @@
+require 'spec_helper'
```

Comment:
Your spec files name need to be suffixed by `_spec` to be run automatically by rspec